tier2_standards/python_version.md

```diff
@@ -1,21 +1,19 @@
 
-- The Python software is fully operational with no errors when invoked from an interpreter that uses Python `3.6` or higher
+- The Python software is fully operational with no errors when invoked from an interpreter that uses the recent Python version(s)
 - Any future contributions to the software must also adhere to the above requirement
-- Any provided environment file(s) in the repository (i.e. `environment.yml`, `requirements.txt`, `setup.py`), must all list a consistent version of Python.
-- Any defined software environments within continuous integration builds (i.e. Jenkins or Travis) must be using the supported Python version(s)
+- Any provided software environment file(s) in the repository (i.e. `environment.yml`, `requirements.txt`, `setup.py`), must all be consitent in their listings of supported Python version(s)
+- Any defined software environments within continuous integration builds (i.e. Jenkins or Travis) must also be using the supported Python version(s)
 
 ## Importance of this standard
 
-With Python being free and open-source, the Python community continues to build and improve the existing language, resulting in periodic releases of new versions of the Python native library as well as many third-party libraries that we commonly use at STScI (e.g. `bokeh`, `astropy`, `numpy`, etc.).  As such, over time the Python software that we write becomes outdated, and the versions of the dependencies that we rely on become unsupported.  This in turn makes it difficult for future developers to maintain existing software and their environments, as well as build new features.  For these reasons, we make an effort to keep our software operational with newer Python versions, as does the Python community as a whole, in order to avoid some of these future issues.
-
-Currently, many of the Python libraries we use support Python versions `3.6`, `3.7`, and/or `3.8`.  As such, we are requiring teams to support _at least_ Python version `3.6`.
+With Python being free and open-source, the Python community continues to build and improve the existing language, resulting in periodic releases of new versions of the Python native library as well as many third-party libraries that we commonly use at STScI (e.g. `bokeh`, `astropy`, `numpy`, etc.).  As such, over time the Python software that we write becomes outdated, and the versions of the dependencies that we rely on become unsupported.  This in turn makes it difficult for future developers to maintain existing software and their environments, as well as build new features.  For these reasons, we make an effort to keep our software operational with newer Python versions, as does the Python community as a whole, in order to avoid some of these future issues.  As such, we are requiring teams to support at least one recent version of Python.  Teams may also choose to support several of the most recent versions.
```

Comment:
We may want to had a sentence, like Jeff said, for developers to keep in mind the person starting from scratch with the newest version of everything and we shouldn't have to ask people to downgrade to use our stuff (within reason, like how python 3.8 is super brand new). Up to you though!